Client/src/Components/Admin/AdminReviewView.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Button, Grid, TextField, Typography, createMuiTheme, withStyles } from '@material-ui/core'
+import { IApprovedDatasetModel, IFlaggedDatasetQuery } from '../../Models/Datasets/IApprovedDatasetModel'
+import React, { useEffect, useState } from 'react'
+import { adminApprovedDataset, callRejectDataset, flagDataset } from '../../Remote/Endpoints/DatasetEndpoints'
+
+import { AdminReviewList } from './AdminReviewList'
+import { DatasetUploadForm } from '../DatasetUpload/DatasetUploadForm'
+
+export function AdminReviewView() {
+
+    const [datasetState, setDatasetState] = useState([])
+    const [editable, setEditable] = useState(false)
+    const [dataset, setDataset] = useState<IApprovedDatasetModel>()
+    const [comment, setComment] = useState("")
+    const [flaggedComment, setFlaggedComment] = useState("")
+    const [update, setUpdate] = useState(0)
+
+    const handleDeleteDataset = async () => {
+        await callRejectDataset(dataset.id)
+        setUpdate(update + 1)
+        handleDatasetChange(null)
+    }
+
+    const handleEditDataset = () => {
+        setEditable(!editable)
+    }
+
+    const handleFlagDataset = async () => {
+        const query: IFlaggedDatasetQuery = { datasetId: dataset.id, flaggedComments: flaggedComment, additionalComments: comment }
```

Comment:
Why is the additional comments being passed when a dataset is flagged?

Reply:
The route is able to take additional comments as a parameter, so they're sent as well.

Reply:
@Arrow105

Reply:
This is part of the stakeholder requirements. He wants the ability to pass those, period.